test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationTests.cs

```diff
@@ -345,5 +364,25 @@ public void NonemptyCollectionSerialization()
             }
             TestUtilities.AssertFailIfErrors(context);
         }
+
+        [Fact]
+        public void NonemptyCollectionSerializationWithStream()
+        {
+            using (MemoryStream stream = new MemoryStream())
```

Comment:
nit, you can do something like

using MemoryStream stream = new();

And avoid indenting the rest of the method here.